### ISSUE TITLE
Titanism

### ISSUE THREAD

**Issue by xirsoi** [xirsoi/ancientreligions#15]
Another heresy idea for Hellenism, the Titan Cult.

Only idea I can think of here is to make it more like the other "barbaric" pagan religions. Perhaps even the ability to go tribal? The basic idea I have in mind is a cult that sees the Olympians as usurpers and disruptive of the natural order. That the Earth is too controlled and needs to be set free.

Or something. Ideas welcome.

**Comment by xirsoi** [comment]
Any ideas? Opinions?

**Comment by jbelig** [comment]
Also good concept. I especially like the fact that your 2 ideas are complete opposites. One wants to move closer to its time, the other even further back to the ancient roots. 
Is it possible to convert to tribal during a game? What happens to all holdings above the 2 permitted?

**Comment by rquinio** [comment]
There's supposedly a new command convert_to_tribal for holdings. But as there is no command to destroy other holdings in the province...

**Comment by xirsoi** [comment]
Are you sure there is no holding destruction command? Sometimes raiding can destroy an entire holding, which just may be an event effect. Could also be hard-coded, of course, but its worth looking into.

**Comment by xirsoi** [comment]
Just had an idea; We could take away looting from hellenism and prometheanism, and add it to the titan cult. Also give the titan cult peace prestige loss and raise_vassal_opinion_loss = no.

Just a few things to make them more like the unreformed pagans to reflect their barbarity. Also, if we exclude hellenic from pagan subjugation, we can make it so Titan Cultists are subject to it/can use it. But, then you might as well play Germanic Pagan, you get more features...

Its a start anyway.

**Comment by jbelig** [comment]
I doesn't matter if you get some more features as germanic, it's mainly the flavor that attracts people here. And don't forget, we're talking about just a heresy.
I'm going to see if I can find the mechanism that destroys holdings while raiding. It's a very neat idea.

**Comment by xirsoi** [comment]
Spending about 10 minutes looking through on_actions brought up zilch. Hopefully you turn up something jbelig.

**Comment by jbelig** [comment]
I was also checking on_actions and the only events related to looting are irrelevant (chest, viking trait etc.)

**Comment by jbelig** [comment]
Found something relevant in defines.lua
Here are the lines:
	LOOT_HOLDING_DEST_MIN_SAFE = 4,
-- there is a 1 in LOOT_HOLDING_DESTRUCTION_ODDS chance that
	LOOT_HOLDING_DESTRUCTION_ODDS = 20,
-- a holding with less than LOOT_HOLDING_DEST_MIN_SAFE buildings will be destroyed by looting
Any idea if those refer somewhere else? Or being in defines.lua means they're hardcoded?

**Comment by xirsoi** [comment]
Being in defines.lua means that its (most likely) part of the engine and we can't do much else with it. Too bad, would have been a really neat and unique mechanic.

Well, I'm sure we can think of other things.

**Comment by jbelig** [comment]
I found the command destroy_landed_title on wiki, but I have no idea how to make it work in our case.
The example is k_asturias = { destroy_landed_title = THIS } 
Is it possible to make it target every holding in a province except the primary one?

**Comment by xirsoi** [comment]
New ideas:

Titan Cultists gain a religious modifier increasing ai zeal and aggression. They should get offensive bonuses, a higher short reign penalty (thinking 4, like the unreformed pagans), raiding, peace prestige loss, no raised vassal opinion penalty, and perhaps even prepared invasions. 

Basically the Titan Cult should be more like the unreformed pagans. 

Titan Cultist life should be violent and a bit chaotic. Their realms should grow fast and fall apart after they die. I'm imagining something like the Wildlings from the Game of Thrones mod.

**Comment by xirsoi** [comment]
I'm super fired up about this, so I'm going to put together a quick proof of concept.

The Titan Cult will have no religious head, and no ability to declare great holy wars, btw. That's way too organized for them ;)

**Comment by xirsoi** [comment]
I'm getting a parse error from Validator on the event file for the titan celebration events (the Thysia). I left a comment on the commit that added the file in the feature/hellenic_heresies branch.
I'd appreciate it if you'd take a look. I haven't been able to figure it out after staring at it for 20 minutes and I think an extra set (or two!) of eyes would help.

**Comment by xirsoi** [comment]
Completed.
